### DISCUSSION TITLE
Reference to avrdude issues not planned -- but PRs will be welcome

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1710] [General]
There are enhancement requests raised or ideas raised, which were deemed to be not the focus of avrdude project. However, if people are really interested and willing to work on it and come out with PRs, then the project may well review the PR and see if it is a good idea to be included.

**Comment by mcuee** [comment]
CH341 related enhancements: nice to have but not the focus

* https://github.com/avrdudes/avrdude/discussions/1441
* https://github.com/avrdudes/avrdude/discussions/1440

AVR910 related enhancements:
* https://github.com/avrdudes/avrdude/discussions/1442
* https://github.com/avrdudes/avrdude/discussions/1449
* https://github.com/avrdudes/avrdude/discussions/1652

linuxgpio enhancement ideas: jtag, PDI and UPDI support. This is nice to have if there are people who are willing to work on them. updi may be more important and more feasible. PDI is less important as ATxmega chips are not that popular. JTAG may be the more difficult one. But then it does not really make sense to go with bitbanged updi if there is a HW UART -- use serialupdi instead. No HW UART -- get a cheap USB to TTL converter. And the performance of such a bitbanged updi implementation will be pretty slow anyway.
* https://github.com/avrdudes/avrdude/discussions/1456 (UPDI)
* https://github.com/avrdudes/avrdude/discussions/1561 (PDI)
* https://github.com/avrdudes/avrdude/discussions/1494 (JTAG)

AVR32 support: 
* https://github.com/avrdudes/avrdude/discussions/1434
* https://github.com/avrdudes/avrdude/discussions/1435

Note: avr32 support issue has been put up in the main issue list as a low priority task.
* https://github.com/avrdudes/avrdude/issues/1882

**Comment by mcuee** [comment]
Other issues.

* https://github.com/avrdudes/avrdude/discussions/1412
* https://github.com/avrdudes/avrdude/discussions/1682
* https://github.com/avrdudes/avrdude/discussions/1676

**Comment by mcuee** [comment]
@stefanrueger and @MCUdude 

I understand that @stefanrueger prefers that we do not keep too many issues and discussions (say <=25 to keep in one page).
@MCUdude thinks it is not worth converting old open issues to Discussions.

I am okay with both ideas and I have closed quite a few dicussions recently. Still I prefer to keep this discussion OPEN to keep track. Hopefully this is okay with both of you.

**Comment by mcuee** [comment]
@stefanrueger and @MCUdude 
Looking at the openning issues, I tend to think we can close some of them as not planned and link here.

Any objections? I actually want to keep #1490 but it seems to be a tough one and no one is working on it now. 

* #1701 
* #1490 
* #1271 
* #1221 
* #1047 
* https://github.com/avrdudes/avrdude/issues/1024
* #883

**Reply by stefanrueger** [reply]
That is actually a great idea to keep a list of neat ideas or known shortfalls, thanks

**Reply by mcuee** [reply]
I have closed 4 of the above 7 issues as `not planned`. We can always reopen them when someone is working on it.
* #1490 
* #1271 
* #1221 (re-open as the root cause was identified) --> fixed by #2128
* #883

I will wait for @MCUdude's comment before closing the remaining three issues..
* #1701 --> closed as not planned
* #1047 --> closed as not planned
* #1024 --> closed as not planned

**Reply by mcuee** [reply]
The following two issues have been closed as "not planned"
* #1047 
* #1024

**Comment by mcuee** [comment]
I have closed the following long standing issues as "not planned" due to the lack of capacity in the team and also that we have a documented workaround.

* https://github.com/avrdudes/avrdude/issues/1253
* https://github.com/avrdudes/avrdude/issues/968

**Reply by mcuee** [reply]
Post avrdude 8.2 release, I may want to take a look at issue #1253 to identify the root cause and see if there is a easy solution or not.
* #1253

As for Issue #968, my idea is to use libusb-1.0 + libusb-compat-0.1.
* #963
* https://github.com/avrdudes/avrdude/issues/2114

**Comment by mcuee** [comment]
* #1221 has been fixed by #2128. It will be included in the upcoming avrdude 8.2 release.

* #1221 
* #2128
